helm-charts/index.yaml

```diff
@@ -0,0 +1,14 @@
+# TODO: update this file after releasing the chart
+apiVersion: v1
+entries:
+  yunikorn:
+  - apiVersion: v1
+    appVersion: 0.1.0
```

Comment:
what does the appVersion mean here?
how is it related to the release version?

Reply:
it takes the appVersion and api version from the Charts.yaml file.
The api version is v1 if we want to support older versions of Helm than v3.
The appVersion should be the version of the application, so I will change it to 0.8.0